src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Broker/IBroker.cs

```diff
@@ -1,4 +1,8 @@
-﻿//----------------------------------------------------------------------
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
```

Comment:
these look like they are in the wrong place

Reply:
Fixed